pkg/renderer/sgml/html5/quoted_string_test.go

```diff
@@ -26,6 +26,14 @@ var _ = Describe("quoted strings", func() {
 </div>`
 			Expect(RenderHTML(source)).To(MatchHTML(expected))
 		})
+		It("simple single quoted unicode", func() {
+			source := ":unicode:\n\n'`curly was single`'"
```

Comment:
same question as below

Reply:
Same answer too... look just above -- I modified the test above by copy paste. :-)

Reply:
yeah, that's what I thought just a few minutes ago 🤦‍♂️   
my apologies